tests/report_status_test.go

```diff
@@ -11,10 +15,78 @@ import (
 	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate.io/v1"
 
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	appsv1client "k8s.io/client-go/kubernetes/typed/apps/v1"
 )
 
+func IsReady(pod corev1.Pod) bool {
+	for _, condition := range pod.Status.Conditions {
+		if condition.Type == corev1.PodReady &&
+			condition.Status == corev1.ConditionTrue {
+			return true
+		}
+	}
+	return false
+}
+
+func WaitPodsCleanup() {
```

Comment:
this should return an error, so we can loudly fail if cleanup fails

Reply:
Done